cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -204,23 +203,16 @@ public void getAppInfo(HttpRequest request, HttpResponder responder,
   @Path("/apps/{app-id}")
   public void deleteApp(HttpRequest request, HttpResponder responder,
                         @PathParam("namespace-id") String namespaceId,
-                        @PathParam("app-id") final String appId)
-    throws BadRequestException, NamespaceNotFoundException {
-
+                        @PathParam("app-id") final String appId) throws Exception {
     Id.Application id = validateApplicationId(namespaceId, appId);
-    AppFabricServiceStatus appStatus;
     try {
       applicationLifecycleService.removeApplication(id);
-      appStatus = AppFabricServiceStatus.OK;
-    } catch (NotFoundException nfe) {
-      appStatus = AppFabricServiceStatus.PROGRAM_NOT_FOUND;
-    } catch (CannotBeDeletedException cbde) {
-      appStatus = AppFabricServiceStatus.PROGRAM_STILL_RUNNING;
-    } catch (Exception e) {
-      appStatus = AppFabricServiceStatus.INTERNAL_ERROR;
+      LOG.trace("Delete call for Application {} at AppFabricHttpHandler", appId);
```

Comment:
Not really. ``AppFabricHttpHandler`` is long gone. But clearly, it has left its trace ;-)